manifests/logrotate.pp

```diff
@@ -0,0 +1,23 @@
+# == Define: heka::logrotate
+#
+# Manages heka log rotation.
+#
+# === Parameters
+#
+# === Examples
+
+# === Authors
+#
+
+define heka::logrotate {
+
+  logrotate::rule { 'heka':
+    path          => '/var/log/hekad.log',
+    rotate        => 5,
+    rotate_every => 'day',
+    size          => '100k',
+    postrotate    => '/etc/init.d/hekad reload',
```

Comment:
/etc/init.d/hekad doesn't exist on el7 and reload action is missing from systemctl. Please use /sbin/service heka restart